common/searchattribute/defs.go

```diff
@@ -81,6 +81,14 @@ const (
 	// define a custom ScheduleId search attribute, in which case the query using the ScheduleId would operate just like
 	// any other custom search attribute.
 	ScheduleID = "ScheduleId"
+
+	// PausedEntities is a search attribute that stores the list paused entities in the workflow.
+	// Format of a single paused entity: "<entity_type>:<entity_id>:<policy_id>".
+	//  * entity_type can be activity, workflow, etc.
+	//  * entity_id is based on type. For activity it will be activity type, for workflow it will be workflow id.
```

Comment:
Maybe call this `primitive_type` and `entity_type` because seems like we'll be using activity type and workflow type in this field?

Reply:
You mean primitive_type  instead of entity_id ? 
`primitive` and `entity` are kind of very close in semantic. And `primitive` usually imply something basic, building block of something. I think the result will be less clear. 
Also for workflow it will be workflow_id or run_id. I don't think we should 'collapse' them (and we also can't).

Reply:
irrelevant.